src/accelerate/accelerator.py

```diff
@@ -231,6 +255,70 @@ def prepare_model(self, model):
             model.forward = torch.cuda.amp.autocast()(model.forward)
         return model
 
+    def _prepare_deepspeed(self, *args):
+
+        ds_plugin = self.state.deepspeed_plugin
+        self.ds_config = ds_plugin.ds_config
+
+        bz = [obj.batch_size for obj in args if hasattr(obj, "batch_size")]
+        assert (
+            len(bz) > 0
+        ), "You must specify a training or evaluation dataloader in `accelerate.prepare()` when using DeepSpeed."
```

Comment:
It think this one would read better as (including the suggested rename form before):
```suggestion
        if len(batch_sizes) == 0:
            raise ValueError("You must specify a training or evaluation dataloader in `accelerate.prepare()` when using DeepSpeed.")
```